ideascube/mediacenter/templates/mediacenter/document_detail.html

```diff
@@ -18,7 +18,7 @@
     {% elif document.preview %}
         <a href="{% media document 'original' %}"><img src="{% media document 'preview' %}" /></a>
     {% endif %}
-    <div class="text">{{ document.summary }}</div>
+    <div class="text">{{ document.summary|safe }}</div>
```

Comment:
Will TinyMCE properly escape HTML entered by a malicious user? (I assume it does, but I prefer asking)

What happens if I disable Javascript and then manually enter some malicious HTML?